kymatio/scattering2d/tests/test_torch_scattering2d.py

```diff
@@ -309,6 +309,8 @@ def test_fft(self, backend):
         y[1, 0, :] = x[0, 0, :] + x[0, 1, :] - x[1, 0, :] - x[1, 1, :]
         y[1, 1, :] = x[0, 0, :] - x[0, 1, :] - x[1, 0, :] + x[1, 1, :]
 
+        y = y/2.0 # renormalization
```

Comment:
Why divide by 2?

Reply:
2^2=4, the FFT is unitary, and now the matrix y=Ax is unitary!

Reply:
Never mind, reading the pytorch documentation the result of the FFT is normalized by dividing by the square root of the product of the first two dimensions.

Reply:
Can we make that clear? E.g. `y /= torch.sqrt(2 ** 2)`. It's a little silly, but more clear, I think.

Reply:
This is now fixed(plz resolve if OK)